src/gfxinit.cpp

```diff
@@ -481,18 +481,28 @@ template <class Tbase_set>
 	if (BaseMedia<Tbase_set>::used_set != nullptr) return true;
 
 	const Tbase_set *best = nullptr;
+
+	auto IsBetter = [&best] (const auto *current) {
+		Debug(sprite, 0, "{} vs {}", (const void*)best, (const void*)current);
```

Comment:
Debug level 0 sounds like a left-over.

Reply:
Indeed. Why didn't I spot that myself in the GitHub interface?